wdl-lint/src/rules/shellcheck.rs

```diff
@@ -64,27 +68,88 @@ static SHELLCHECK_EXISTS: OnceLock<bool> = OnceLock::new();
 /// The identifier for the command section ShellCheck rule.
 const ID: &str = "ShellCheck";
 
+#[derive(Clone, Debug, Deserialize)]
+/// Suggested fix for a ShellCheck diagnostic.
+struct ShellCheckFix {
+    /// The replacements to perform.
+    pub replacements: Vec<ShellCheckReplacement>,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+/// A ShellCheck replacement.
+///
+/// This differs from a [`Replacement`] in that
+/// 1) columns are 1-indexed
+/// 2) it may span multiple lines and thus cannot be directly passed to a
+///    [`Fixer`].
+///
+/// It must be normalized with `normalize_replacements` before use.
+struct ShellCheckReplacement {
+    /// Line number replacement occurs on.
+    pub line: usize,
+    /// Line number replacement ends on.
+    #[serde(rename = "endLine")]
+    pub end_line: usize,
+    /// Order in which replacements should happen. Highest precedence first.
+    pub precedence: usize,
+    /// An `InsertionPoint`.
+    #[serde(rename = "insertionPoint")]
+    pub insertion_point: InsertionPoint,
+    /// Column replacement occurs on.
+    pub column: usize,
+    /// Column replacements ends on.
+    #[serde(rename = "endColumn")]
+    pub end_column: usize,
+    /// Replacement text.
+    pub replacement: String,
```

Comment:
```suggestion
    pub value: String,
```
to match the core `Replacement` struct